mix.exs

```diff
@@ -35,11 +35,11 @@ defmodule Membrane.Element.Opus.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.4.1"},
-      {:membrane_common_c, "~> 0.2.1"},
-      {:membrane_caps_audio_raw, "~> 0.1"},
-      {:membrane_caps_audio_opus, github: "membraneframework/membrane-caps-audio-opus"},
-      {:unifex, "~> 0.2.3"},
+      {:membrane_core, "~> 0.5.0"},
+      {:membrane_opus_format, "~> 0.2.0"},
+      {:membrane_caps_audio_raw, "~> 0.2"},
+      {:membrane_common_c, "~> 0.3.0"},
```

Comment:
Nitpick: do we use it directly? Can we remove common_c from explicit deps list?

Reply:
We do `#include <membrane/log.h>`.  There's no native logging now, but I'd keep the dep to make adding it straightforward